add_book.php

```diff
@@ -0,0 +1,34 @@
+<?php
```

Comment:
Observations: Including HTML and PHP in the same file can make the code harder to maintain and read.

Recommendations:
- Separate the business logic from the presentation layer by using a templating system or separating the PHP logic into a controller. This will help in maintaining a clear separation of concerns.